soroban-env-host/src/host.rs

```diff
@@ -199,6 +199,17 @@ impl Host {
         }
     }
 
+    #[cfg(feature = "testutils")]
+    pub fn with_mutable_ledger_info<F>(&self, mut f: F) -> Result<(), HostError>
```

Comment:
The shorter term is used frequently in functions for mutable variables.
```suggestion
    pub fn with_mut_ledger_info<F>(&self, mut f: F) -> Result<(), HostError>
```